e2e/test/scenarios/embedding/embedding-native.cy.spec.js

```diff
@@ -81,62 +82,61 @@ describe("scenarios > embedding > native questions", () => {
         assert.deepEqual(actual, expected);
       });
 
-      visitIframe();
+      cy.get("@questionId").then(questionId => {
+        const payload = {
+          resource: { question: questionId },
+          params: { total: [] },
+        };
 
-      // eslint-disable-next-line no-unscoped-text-selectors -- deprecated usage
-      cy.contains("Organic");
-      // eslint-disable-next-line no-unscoped-text-selectors -- deprecated usage
-      cy.contains("Twitter").should("not.exist");
+        visitEmbeddedPage(payload);
```

Comment:
This is a good change, but we shouldn't nest all the assertions unnecessarily.

Once we finished using `questionId` we should close the scope, so all the assertions should be left as is.

```ts
      cy.get("@questionId").then(questionId => {
        const payload = {
          resource: { question: questionId },
          params: { total: [] },
        };

        visitEmbeddedPage(payload);
      }) // close the block here.
```